### DISCUSSION TITLE
Crepuscolare complesso

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#269] [HUB: Home Assistant]
Salve, mi trovo a dover risolvere un problema che sono sicuro che anche altri hanno avuto, mi chiedevo quale sia il modo migliore per risolverlo.
Devo installare in giardino 1 faretto con sensore di movimento+crepuscolare, in modo che quando la sera arrivo con la macchina mi illumini il parcheggio.
Vorrei inoltre la possibilità all'occorrenza di accenderlo a prescindere, magari mi serve la luce o devo vedere qualcosa.....
Vorrei la possibilità di spegnerlo a prescindere (magari anche per fasce orarie, ad esempio di notte non vorrei che passa un gatto e si accende il faretto alle 2 di notte).
Detto cià ho trovato soluzioni solo con un crepuscolare + almeno 2 interruttori (a prescindere se smart o meno) a meno di non avere una centralina più complessa che gestisca il tutto...
Mi ci vorrebbero 2 interruttori smart: uno per forzare l'accensione del faretto, messo in parallelo al crepuscolare, ed uno per spegnere tutto comunque....
Mi date qualche idea?

PS: ovvaimete se lo controlal da HA vorrei evitare di avere 2 pulsanti 1 sempre acceso, che se lo spengo spengo etutto e uno sempre spento che se lo accendo accendo tutto. Metterei un pulsante a 3 tacche che a sinistra è TUTTO SPENTO, al centro decide il crepuscolare a  destra è sicuramente acceso....

**Comment by barbatrukko** [comment]
Nessuna idea? Nessuno ha mai avuto questa esigenza?

**Reply by gforment** [reply]
Io ho una luce esterna:
- Accensione/spegnimento: la luce è collegata ad un Outdoor Plug della Meross (con tasto fisico che fa da backup)
- Rilevatore di movimento: Aqara Presence
- Interruttori: Aqara Opple
Il tutto gestito tramite HA, Z2M e Mosquitto.
Le automazioni prevedono spegnimento/accensione dai tasti (Opple) e accensione se c'è movimento (Presence) dopo che il sole è tramontato e non sia già accesa+spegnimento dopo 5min. Coi gatti qualche volta accade si accenda ma non è un dramma.
Spero di aver capito correttamente la tua esigenza :)
